Protocol.md

```diff
@@ -198,17 +226,23 @@ Frame Contents
      0                   1                   2                   3
      0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-    |R|                    Frame Length (optional)                  |
-    +-------------------------------+-+-+-+-+-----------------------+
-    |     Frame Type = SETUP        |0|M|L|S|       Flags           |
-    +-------------------------------+-+-+-+-+-----------------------+
-    |                          Stream ID = 0                        |
-    +-------------------------------+-------------------------------+
+    |                           Stream ID                           |
```

Comment:
That's a pretty interesting point. The positive is certainly that it will reduce the data transferred for leases, keep-alive but the negative would be inconsistency and bugs related to missing stream ID counted as stream 0. May be hard to debug these bugs.